pkgs/sentry/default.nix

```diff
@@ -2,6 +2,12 @@
 
 let
   python = import ./requirements.nix { inherit pkgs; };
+
+  django_1_11_patch = pkgs.fetchpatch {
```

Comment:
This is bringing back fond memories of Django - not!

Reply:
Save me